File: 268-missing-number/missing-number.cpp
class Solution {
public:
    int missingNumber(vector<int>& nums) {
        int missing=-1;
        sort(nums.begin(),nums.end());
        for(int i=0; i<nums.size(); i++)
        {
            if(nums[i]!=i)
            {
                missing=i;
                break;
            }
        }
        if(missing==-1)
        {
            missing=nums.size();
        }
        return missing;
    }
};

File: 268-missing-number/missing-number.py
class Solution:
    def missingNumber(self, nums: List[int]) -> int:
        length = len(nums)
        maxlength = (length*(length+1))//2
        sumval = sum(nums)
        #print(sumval)
        #print(maxlength)
        return (maxlength-sumval)